python/ray/serve/_private/deployment_state.py

```diff
@@ -396,6 +396,12 @@ def log_file_path(self) -> Optional[str]:
         """Returns the relative log file path of the actor, None if not placed."""
         return self._log_file_path
 
+    @property
+    def initialization_latency_s(self) -> float:
+        """Returns the initialization latency for the replica actor."""
```

Comment:
note that it returns `None` until the replica is initialized (or add an assertion that it's only called after)

Reply:
btw, what happens to this field if the controller crashes and we go through the recovery path?

Reply:
in this case does the the log statement need to be made conditional to avoid logging `None` in some edge cases?

Reply:
I changed the implementation to save the initialization latency on the replica side. It gets reported back to the controller when the controller recovers, so we should never log `None`. However, I made the log statement conditional just to be safe.